.env.sample

```diff
@@ -1,5 +1,10 @@
-PANO_MINIO_URL=127.0.0.1:9000
-PANO_MINIO_ACCESS_KEY=zlGjx3eZm9JLEc39A87f
-PANO_MINIO_SECRET_KEY=yPhvV67qhUItegOSTdKbhX1eerZSHe86SciwJ6Td
-PANO_MINIO_BUCKET=panoramas
-PANO_MINIO_SECURE=False
+PG_CONN=postgresql://pano-dev:pano-dev@localhost:5432/pano-dev
+
+MINIO_URL=127.0.0.1:9000
+MINIO_ACCESS_KEY=zlGjx3eZm9JLEc39A87f
+MINIO_SECRET_KEY=yPhvV67qhUItegOSTdKbhX1eerZSHe86SciwJ6Td
+MINIO_BUCKET=panoramas
+MINIO_SECURE=False
+
+MESHDB_ENDPOINT=https://example.com/
```

Comment:
This could default to http://127.0.0.1:8000. That's where meshdb usually runs when you run it on baremetal.